src/sails_forth/http.clj

```diff
@@ -66,10 +66,13 @@
 (s/def ::access_token
   string?)
 
-(s/def ::authentication
+(s/def ::non-nil-authentication
```

Comment:
The reason for these acrobatics is that `::state` uses an `:authentication` entry, and in that context it's nilable. So this is the only combination that seems to work without using another namespace prefix.